src/Workspaces/Core/Portable/Log/WorkspaceErrorLogger.cs

```diff
@@ -21,7 +21,7 @@ public WorkspaceErrorLogger()
         }
 
         public void LogException(object source, Exception exception)
-            => Logger.GetLogger()?.Log(FunctionId.Extension_Exception, LogMessage.Create(source.GetType().Name + " : " + ToLogFormat(exception), LogLevel.Error));
+            => Logger.Log(FunctionId.Extension_Exception, () => source.GetType().Name + " : " + ToLogFormat(exception), LogLevel.Error);
```

Comment:
📝 This lambda is still allocating. The easiest way to fix is to switch to a `static` lambda and use the `Log` overload that allows arguments to be passed (in this case, `source`, `exception`, and possibly `this`).